addon/components/table-of-contents-plugin/ember-nodes/table-of-contents.ts

```diff
@@ -89,13 +89,13 @@ export default class TableOfContentsComponent extends Component<EmberNodeArgs> {
         );
         const config = this.config[0];
         if (config.scrollContainer) {
-          const scrollContainer: HTMLElement = config.scrollContainer;
+          const scrollContainer: HTMLElement = config.scrollContainer();
           const alreadyScrolled = scrollContainer.scrollTop;
-          const MAGIC_NUMBER_TOPBAR_HEIGHT: number =
-            config.scrollingPadding ?? 150;
           scrollContainer.scrollTo(
             0,
-            coords.top + alreadyScrolled - MAGIC_NUMBER_TOPBAR_HEIGHT
+            coords.top +
+              alreadyScrolled -
+              (scrollContainer.getBoundingClientRect().y + 10)
```

Comment:
I'd make this less magic-number (the +10 mostly).
Could maybe add `const beginHeightAndPadding = scrollContainer.getBoundingClientRect().y + 10` or just `const topPadding = 10` or even just a comment.
This is a bit of a "magic" solution, so some comment or at least code that explains itself would be nice :)

Reply:
Done :)